packages/webapp/pages/squads/create.tsx

```diff
@@ -191,7 +191,7 @@ function CreatePost(): ReactElement {
         <TabContainer<WriteFormTab>
           onActiveChange={(active) => setDisplay(active)}
           controlledActive={display}
-          shouldMountInactive={false}
+          shouldMountInactive={true}
```

Comment:
> By default, shouldMountInactive is set to false. This means that when a tab is inactive, its content is unmounted from the DOM. So I have come up with this fix. But not sure if this is a good approach or not.

Normally we do this just to be more optimized than having the elements rendered in the background, but in cases where we need the state to be consistent, we turn this to `true`. So this means, the approach was not wrong.

Reply:
Yes, I thought so if there were many tabs then it would be slow to keep this always true. But in the case of just two tabs, this shouldn't be slow that was my thinking

Reply:
Yeah, this should be good enough for this fix 🚀